expression/builtin_time.go

```diff
@@ -1760,7 +1768,74 @@ type builtinConvertTzSig struct {
 // eval evals a builtinConvertTzSig.
 // See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_convert-tz
 func (b *builtinConvertTzSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("CONVERT_TZ")
+	args, err := b.evalArgs(row)
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+
+	if args[0].IsNull() || args[1].IsNull() || args[2].IsNull() {
+		return
+	}
+
+	sc := b.ctx.GetSessionVars().StmtCtx
+	fsp := types.DateFSP(args[0].GetString())
```

Comment:
MySQL also accept the first argument as an integer, not only a string.
mysql> SELECT CONVERT_TZ(20040101,'+00:00','+10:00');
+----------------------------------------+
| CONVERT_TZ(20040101,'+00:00','+10:00') |
+----------------------------------------+
| 2004-01-01 10:00:00                    |
+----------------------------------------+
1 row in set (0.00 sec)

mysql>